buildenv.mk

```diff
@@ -58,7 +58,8 @@ LINUX_UNITTESTS       := $(ROOT_DIR)/unittests
 DCAP_DIR              := $(LINUX_EXTERNAL_DIR)/dcap_source
 LIBUNWIND_DIR         := $(ROOT_DIR)/sdk/cpprt/linux/libunwind
 
-CP    := /bin/cp -f
+#CP    := /bin/cp -f
```

Comment:
```suggestion
```

I'd suggest to just delete that line instead of commenting it out. If `mkdir` works, `cp` should also work as both are usually part of [`coreutils`](https://www.gnu.org/software/coreutils/).

Reply:
Thanks! Forgot that somehow. Removed and rebased.